Code/GraphMol/Wrap/props.hpp

```diff
@@ -77,13 +77,13 @@ boost::python::dict GetPropsAsDict(const T &obj,
   // std::vector<int>, std::vector<unsigned>, string
   STR_VECT keys = obj.getPropList(includePrivate, includeComputed);
   for(size_t i=0;i<keys.size();++i) {
-    if (AddToDict<double>(obj, dict, keys[i])) continue;
     if (AddToDict<int>(obj, dict, keys[i])) continue;
     if (AddToDict<unsigned int>(obj, dict, keys[i])) continue;
```

Comment:
Looks like boost figures out the signed unsigned types ok, python doesn't make a distinction here.  We should really try on 32 bit though... sigh...

```
>>> m = Chem.MolFromSmiles("C")
>>> m.SetProp("max unsigned", "4294967295")
>>> m.SetProp("negative 1", "-1")
>>> m.SetProp("max unsigned + 1", "4294967296")
>>> m.GetPropsAsDict()
{'max unsigned + 1': 4294967296.0, 
  'negative 1': -1, 
  'max unsigned': 4294967295}
```